### DISCUSSION TITLE
Vercel blocked my CDN's requests

### DISCUSSION THREAD

**Discussion by Nemmcy** [vercel/vercel#8854] [Help]
I use Bunny which is an excellent CDN, but it seems that Vercel's firewall blocked their IP addresses according to Bunny's team. Everything worked fine all day and all of a sudden only 502 responses.

"access-control-allow-origin: *" is already set, are there other headers that might help or a way to whitelist all the IPs? There's no way to whitelist IPs on Vercel's dashboard...

**Comment by amyegan** [comment]
Sorry you're running into this problem @nemdo. Are you still getting the 502 error? 

I'm not aware of Vercel blocking Bunny CDN specifically, but I would like to take a look at the logs and get more information on what is happening. Can you share a link to the website or specific deployment so I can find the right logs?
